cmd/launcher/svc_config_windows.go

```diff
@@ -456,3 +458,113 @@ func checkRootDirACLs(logger *slog.Logger, rootDirectory string) {
 
 	logger.Log(context.TODO(), slog.LevelInfo, "updated ACLs for root directory")
 }
+
+func checkEnrollSecretACLs(logger *slog.Logger, enrollSecretPath string) {
+	logger = logger.With(
+		"component", "checkEnrollSecretACLs",
+		"enroll_secret_path", enrollSecretPath,
+	)
+
+	if strings.TrimSpace(enrollSecretPath) == "" {
+		logger.Log(context.TODO(), slog.LevelError,
+			"unable to check permissions without enroll secret path set, skipping",
+		)
```

Comment:
I would do a debug level here, or at least not an error level -- it's not really an error we can do something about, and once we support secretless it won't be an error at all